README.md

```diff
@@ -1 +1,10 @@
-# hello-microverse
\ No newline at end of file
+# Hello Microverse! Project
+>This project is made as a practise for HTML and CSS, and professional code writing.
+
+## Built with 
+- HTML
+- CSS
+
+## Author
+**Paul Kaizirege**
+- Github : [@kaizipaul]
```

Comment:
- [ ] You didn't add all the fields from this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) in your readme file such as **Contributing**, **Show your support**, **Acknowledgement**. This is your readme file so please take it seriously :blush: